src/js/component.js

```diff
@@ -1398,14 +1399,26 @@ class Component {
    * @method getComponent
    */
   static getComponent(name) {
-    if (Component.components_ && Component.components_[name]) {
-      return Component.components_[name];
+    if (!name) {
```

Comment:
In the old code and the new code, a missing `Component.components_` or a missing `Component.components_[name]` will return `undefined` - the only difference is the old code was simpler. It seems like this block should go back to its old state, but with a `name = toTitleCase(name);` line before it:

```js
name = toTitleCase(name);
if (Component.components_ && Component.components_[name]) {
  return Component.components_[name];
}
```